exo-go/src/app_config_helpers/index_test.go

```diff
@@ -0,0 +1,137 @@
+package appConfigHelpers_test
+
+import (
+	"os"
+	"path"
+
+	"github.com/Originate/exosphere/exo-go/src/app_config_helpers"
+	"github.com/Originate/exosphere/exo-go/src/types"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var (
+	appConfig types.AppConfig
+	services  types.Services
+)
+
+var _ = BeforeSuite(func() {
+	exocom := types.Dependency{Name: "exocom", Version: "0.22.1", Silent: true}
+	mongoConfig := types.DependencyConfig{DependencyEnvironment: map[string]string{"DB_NAME": "test-db"}}
+	mongo := types.Dependency{Name: "mongo", Version: "3.4.0", Config: mongoConfig}
+	dependencies := []types.Dependency{exocom, mongo}
+
+	todoService := types.ServiceData{Location: "./todo-service", NameSpace: "todo"}
+	externalService := types.ServiceData{DockerImage: "originate/test-web-server", Silent: true}
+	services = types.Services{
+		Private: map[string]types.ServiceData{"todo-service": todoService},
+		Public:  map[string]types.ServiceData{"external-service": externalService},
+	}
+
+	appConfig = types.AppConfig{
+		Name:         "test ",
+		Version:      "0.1.0",
+		Services:     services,
+		Dependencies: dependencies,
+	}
+})
+
+var _ = Describe("GetAppConfig", func() {
+	var appConfig types.AppConfig
+
+	It("should read application.yml successfully", func() {
+		appDir := path.Join("..", "..", "..", "exosphere-shared", "example-apps", "complex-setup-app")
+		if err := os.Chdir(appDir); err != nil {
+			panic(err)
+		}
+		var err error
+		appConfig, err = appConfigHelpers.GetAppConfig()
+		Expect(err).NotTo(HaveOccurred())
+	})
+
+	It("should include name, version and description", func() {
+		Expect(appConfig.Name).To(Equal("complex-setup-app"))
+		Expect(appConfig.Description).To(Equal("An app with complex setup used for testing"))
+		Expect(appConfig.Version).To(Equal("0.0.1"))
+	})
+
+	It("should have all the dependencies", func() {
+		exocom := types.Dependency{Name: "exocom", Version: "0.22.1"}
+		mongoConfig := types.DependencyConfig{
+			Ports:                 []string{"4000:4000"},
+			Volumes:               []string{"{{EXO_DATA_PATH}}:/data/db"},
+			OnlineText:            "waiting for connections",
+			DependencyEnvironment: map[string]string{"DB_NAME": "test-db"},
+			ServiceEnvironment:    map[string]string{"COLLECTION_NAME": "test-collection"},
+		}
+		mongo := types.Dependency{Name: "mongo", Version: "3.4.0", Config: mongoConfig}
+		expected := []types.Dependency{exocom, mongo}
```

Comment:
Can we remove the exocom and mongoConfig variables? I think we can even remove the expected variable.
```go
Expect(appConfig.Dependencies).To(Equal([]types.Dependency{
  types.Dependency{
    Name: "exocom", 
    Version: "0.22.1",
  },
  types.Dependency{
    Name: "mongo", 
    Version: "3.4.0", 
    Config: types.DependencyConfig{
      Ports:                 []string{"4000:4000"},
      Volumes:               []string{"{{EXO_DATA_PATH}}:/data/db"},
      OnlineText:            "waiting for connections",
      DependencyEnvironment: map[string]string{"DB_NAME": "test-db"},
      ServiceEnvironment:    map[string]string{"COLLECTION_NAME": "test-collection"},
    },
  },
}))
```

A similar idea can be applied to other tests